source/_patterns/01-molecules/teasers/teaser.mustache

```diff
@@ -5,7 +5,7 @@
   {{#contextLabel}}
     <ol class="teaser__context_label_list" aria-label="These research categories are for the following article">
       {{#list}}
-        <li class="teaser__context_label_item">{{#url}}<a href="{{url}}" {{#contextAriaLabel}}aria-label="{{contextAriaLabel}}" {{/contextAriaLabel}}{{#carouselItem}}tabindex="-1" {{/carouselItem}}class="teaser__context_label_link">{{/url}}{{{name}}}{{#url}}</a>{{/url}}</li>
+        <li class="teaser__context_label_item">{{#url}}<a href="{{url}}" {{#ariaLabel}}aria-label="{{ariaLabel}}" {{/ariaLabel}}{{#carouselItem}}tabindex="-1" {{/carouselItem}}class="teaser__context_label_link">{{/url}}{{{name}}}{{#url}}</a>{{/url}}</li>
```

Comment:
Just thinking... What are we going to put in this aria-label?

Reply:
We agreed contextAriaLabel is not needed so I removed it. The headerAriaLabel is renamed to just ariaLabel.